study.md

```diff
@@ -27,17 +27,23 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+<!-- The DOM is not a source or a rendered page.
+
+source : the README file in this repo. -->
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+<!-- Yes! Because developer tools are a visual representation of the DOM.
+
+souce : https://css-tricks.com/dom/ -->
 ```
```

Comment:
Correct that what you see in dev tools is a visual representation of the DOM. This was kind of a trick question. I'd say kind of rather than yes only because they are not the same